m0t1fs/linux_kernel/file.c

```diff
@@ -1593,13 +1592,15 @@ static int pargrp_iomap_parity_recalc(struct pargrp_iomap *map)
 				old[col]   = map->pi_databufs[row][col]->
 					db_auxbuf;
 
-				m0_parity_math_diff(parity_math(map->pi_ioreq),
-						    old, dbufs, pbufs, col);
+				rc = m0_parity_math_diff(parity_math(map->pi_ioreq),
```

Comment:
The original "m0_parity_math_diff" returns void. 
We reverted some functions to return void as original.
How about this?

Reply:
As discussed [in this comment](https://github.com/Seagate/cortx-motr/pull/667#discussion_r652685154) we have allocation of buffer `diff_data_arr` in reed_solomon_diff(). Hence I have kept return type as int to return m0_alloc failures if any. Also this buffer can not be allocated in init because it is of size block_size, which is not known during init.

Reply:
OK. I see.